src/test/java/org/broadinstitute/hellbender/tools/longreads/AlignedReadsToGraphConverterIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.tools.longreads;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class AlignedReadsToGraphConverterIntegrationTest extends CommandLineProgramTest {
+
+    private static final String aligned_bam_file = toolsTestDir + "longreads" + File.separator + "NA19240.chr20_19294351-19304184.corrected.bam";
+
+    @Test
+    public void testSerializeToDotFile() {
+
+        final File tmpDirName = createTempDir("alignedReadsGraphTest");
```

Comment:
These tests should assert something about the output (even if its just against a checked in result file)